src/Sentry/Transaction.cs

```diff
@@ -150,28 +150,28 @@ public IReadOnlyList<string> Fingerprint
         }
 
         // Not readonly because of deserialization
-        private Lazy<ConcurrentBag<Breadcrumb>> _breadcrumbsLazy = new();
+        private ConcurrentBag<Breadcrumb> _breadcrumbs = new();
 
         /// <inheritdoc />
-        public IReadOnlyCollection<Breadcrumb> Breadcrumbs => _breadcrumbsLazy.Value;
+        public IReadOnlyCollection<Breadcrumb> Breadcrumbs => _breadcrumbs;
 
         // Not readonly because of deserialization
-        private Lazy<ConcurrentDictionary<string, object?>> _extraLazy = new();
+        private ConcurrentDictionary<string, object?> _extra = new();
```

Comment:
Wouldn't say it's the same as `Lazy` at all:

https://github.com/microsoft/referencesource/blob/master/mscorlib/system/collections/Concurrent/ConcurrentDictionary.cs#L122